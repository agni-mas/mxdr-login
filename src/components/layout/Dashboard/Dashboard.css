.dashboard {
    /* min-height: 500px; */
}