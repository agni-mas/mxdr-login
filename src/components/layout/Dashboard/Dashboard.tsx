import { withEmotionCache } from '@emotion/react';
import { whitelabel_configuration } from '@looker/sdk';
import React, { FC, useEffect, useState } from 'react';
import './Dashboard.css';

interface DashboardProps {
  children: any,
  name: string
}

const styles = {
  greeting: {
    fontFamily: 'Graphik',
    fontStyle: 'normal',
    fontWeight: 300,
    fontSize: 26,
    paddingBottom: 19,
    color: 'white',
  }
}

const Dashboard: FC<DashboardProps> = ({ children, name = 'Alex' }) => {

  return (
    <section className='dashboard'>
      <div style={styles.greeting}>Good day, {name}</div>
      {children}
    </section>
  );
};

export default Dashboard