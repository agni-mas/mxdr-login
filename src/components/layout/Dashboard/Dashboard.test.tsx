import { render } from "@testing-library/react";
import Dashboard from "./Dashboard";

it("renders <Dashboard/>", () => {
    const props = {
        children: <div>Some Chart Component</div>,
        name: 'Alex'
    }

    const { asFragment } = render(<Dashboard children={props.children} name={props.name} />);
    expect(asFragment()).toMatchSnapshot();
});