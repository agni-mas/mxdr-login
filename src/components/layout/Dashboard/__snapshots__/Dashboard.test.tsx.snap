// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders <Dashboard/> 1`] = `
<DocumentFragment>
  <section
    class="dashboard"
  >
    <div
      style="font-family: Graphik; font-style: normal; font-weight: 300; font-size: 26px; padding-bottom: 19px; color: white;"
    >
      Good day, Alex
    </div>
    <div>
      Some Chart Component
    </div>
  </section>
</DocumentFragment>
`;
